RetrospectiveExtension.Frontend/vss-extension-dev.json

```diff
@@ -1,11 +1,12 @@
 {
   "manifestVersion": 1,
   "id": "retrospective-vsts-extension-dev",
-  "publisher": "EnginPolat",
-  "version": "1.0.260",
+  "publisher": "NeyissaExilus",
+  "version": "1.0.264",
```

Comment:
it reminds me, I should remove my name from this file. could we replace my name with a placeholder? like; <publisher_name> or something like that?

Reply:
@polatengin, I believe the `vss-extension-dev.json.template` file below is supposed to serve that purpose. I don't think file was meant to be checked in

Reply:
@focusthenflex , that makes sense. can we remove this file from the repo, if it's not removed already, please?